lib/task/arCacheDescriptionXmlTask.class.php

```diff
@@ -57,7 +58,7 @@ private function exportAll($options)
     $logger->log('Caching XML representations of information objects...');
 
     $cache = new QubitInformationObjectXmlCache(array('logger' => $logger));
-    $cache->exportAll(array('skip' => $options['skip'], 'limit' => $options['limit']));
+    $cache->exportAll(array('skip' => $options['skip'], 'limit' => $options['limit'], 'format' => $options['format']));
```

Comment:
I think the other options won't affect the `exportAll` function so it may be better to use `$options` directly. Not a big deal for sure.